corehq/apps/hqadmin/reports.py

```diff
@@ -495,3 +498,64 @@ def headers(self):
     @property
     def rows(self):
         return self.table_data.rows
+
+
+class StaleCasesTable:
+
+    STALE_DATE_THRESHOLD_DAYS = 365
+
+    @property
+    def headers(self):
+        return DataTablesHeader(
+            DataTablesColumn(gettext_lazy("Domain")),
+            DataTablesColumn(gettext_lazy("Case count"))
+        )
+
+    @property
+    def rows(self):
```

Comment:
Using `table.rows` twice will run the query twice. How about something like this?
```python
    def __init__():
        self._rows = None

    @property
    def rows(self):
        if self.rows is None:
            self._rows = []
            # etc.
        return self._rows
```

Reply:
I like this idea! Currently the "caching" is done in the task itself by calling `table_rows = StaleCasesTable().rows` and then using the `table_rows` var locally, but the above suggestion will remove the need for storing the rows locally in the task.